crates/swc_css_minifier/tests/fixture/compress-calc/units/output.min.css

```diff
@@ -0,0 +1 @@
+.class1{width:calc(1px + 1)}.class2{width:calc(100% - 60px)}.class3{width:calc(100% - -10px)}.class4{width:950ms}.class5{width:calc(99.99%*1/1 - 0rem)}.class6{width:calc(50% + (5em + 5%))}.class7{width:calc((100px - 1em) + (-50px + 1em))}
```

Comment:
`width:calc(100% - -10px)` -> `width:calc(100% + 10px)`

Reply:
`width:calc((100px - 1em) + (-50px + 1em))` -> `width:50px`, but I see `TODO`, so we can postpone it

Reply:
This case is now handled :)